.NET/Microsoft.Recognizers.Definitions.Common/Turkish/DateTimeDefinitions.cs

```diff
@@ -21,6 +21,7 @@ namespace Microsoft.Recognizers.Definitions.Turkish
 
     public static class DateTimeDefinitions
     {
+      public const bool CheckBothBeforeAfter = true;
```

Comment:
Does Turkish really needs to check "both sides" in all these situations? I'm wondering if we should have finer-grained flags.

Reply:
This seems the simpler solution at the moment, since the same methods are used on patterns that have different structures (match in the prefix or match in the suffix).
For example, "iki bin yılı Eylül'ün onunda" (on the tenth of September, two thousand) and "10 eylül bin dokuz yüz bir" (on September 10, nineteen zero one) are both parsed by ParseNumberWithMonth. But in the first, the year "iki bin yılı" is found in the prefix, while in the second the year "bin dokuz yüz bir" is found in the suffix.